### ISSUE TITLE
Possible Step 1 Oracle VM VirtualBox installation improvement

### ISSUE THREAD

**Issue by hkcyf369** [open-learning-exchange/open-learning-exchange.github.io#2472]
### Problem
For Windows 10 users, in order to successfully install and use the VirtualBox, it is necessary to enable the Intel virtualization technology. 

### Steps to reproduce the problem

N/A

### Screenshots

N/A

### Proposed solution

In Step1 "Troubleshooting", add this video (for Windows users):
https://www.youtube.com/watch?v=KxYaDQvJizU

**Comment by hemu-codes** [comment]
Could you maybe include some images of possible error messages that the installation gives to Windows users - because right now, it seems very vague and new users might not be entirely on when to use this solution.

**Comment by SNutakki** [comment]
I agree with hemu. Please add more details to your proposed solution and we can see if it's satisfactory.

**Comment by hkcyf369** [comment]
Sure. I will add a screenshot. Thanks for the comments.

**Comment by ChiragAwale** [comment]
<img width="925" alt="Screen Shot 2019-06-27 at 9 15 57 AM" src="https://user-images.githubusercontent.com/19680263/60273714-52af3b00-98bc-11e9-9aba-cb94180ebe82.png">

**Comment by ChiragAwale** [comment]
This is in step 1. I do not think we need to add it.

**Comment by hkcyf369** [comment]
Oh, I'm sorry, I miss the prerequisites part. However, maybe we can add a line saying that:

> If you see an error like this : 
<img width="513" alt="VM_Error" src="https://user-images.githubusercontent.com/47688356/60330824-250fd380-9948-11e9-9a5f-035e4a0887c0.PNG">

> It is likely that your virtualization is disabled, please come back and make sure the virtualization is on.

**Comment by Liew211** [comment]
@hkcyf369 I think that would be better.  You can add this in the Windows troubleshooting section at the end.

**Comment by lmmrssa** [comment]
@hkcyf369 virtualization needs to be enabled in all OS if you have not enabled it on BIOS. Also each laptop manufacture has different BIOS configuration. Also video you proposed does not show anything about VT-x its is just showing how to get to BIOS.

**Comment by biswasshrestha** [comment]
I agree that the currently available instructions that state the importance of enabling virtualization in the BIOS are enough and I do not think it will be valuable to add error messages that can vary for each user.
